pkg/virtctl/vm/vm.go

```diff
@@ -193,6 +242,45 @@ func (o *Command) Run(cmd *cobra.Command, args []string) error {
 		if err != nil {
 			return fmt.Errorf("Error renaming VirtualMachine %v", err)
 		}
+	case COMMAND_GUESTOSINFO:
+		guestosinfo, err := virtClient.VirtualMachineInstance(namespace).GuestOsInfo(vmiName)
+		if err != nil {
+			return fmt.Errorf("Error renaming VirtualMachine %v", err)
```

Comment:
The message is wrong here.
This seems to be repeated in the rest of the changes below.